services/brig/src/Brig/Options.hs

```diff
@@ -353,6 +353,8 @@ data Opts = Opts
     elasticsearch :: !ElasticSearchOpts,
     -- | AWS settings
     aws :: !AWSOpts,
+    -- | Enable Random Prekey Strategy
+    randomPrekeys :: !(Maybe Bool),
```

Comment:
(I would like this more if it was a `Bool` and would parse an optional json boolean with `fromMaybe False`, but I think your pattern is the one we're using in other places, too.)

Reply:
Yes I wondered what was the cultural pattern :) In the end having the Env be Bool, but the Opt be Maybe Bool seemed like an ok choice to me.